website/source/docs/builders/profitbricks.html.md

```diff
@@ -0,0 +1,74 @@
+---
+description: |
+    The ProfitBricks builder is able to create images for ProfitBricks cloud.
+layout: docs
+page_title: ProfitBricks Builder
+...
+
+# ProfitBricks Builder
+
+Type: `profitbricks`
+
+The ProfitBricks Builder is able to create virtual machines for [ProfitBricks](https://www.profitbricks.com).
+
+-> **Note:** This builder is not supported by HashiCorp. Please visit [ProfitBricks DevOps Central](https://devops.profitbricks.com/) for support. You may file issues on [GitHub](https://github.com/profitbricks/packer-builder-profitbricks/issues).
```

Comment:
@mwhooker I think this is not a good way to handle this. I think ProfitBricks and HashiCorp should sort out how to handle maintenance.